#! /bin/sh -e

# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: List et_EE with charset ISO-8859-15 as a supported locale
# DP: Related bugs: #208238
# DP: Dpatch author: Petter Reinholdtsen
# DP: Patch author: Petter Reinholdtsen
# DP: Upstream status: In CVS
# DP: Status Details: http://sources.redhat.com/bugzilla/show_bug.cgi?id=118
# DP: Date: 2004-04-18

PATCHLEVEL=0

if [ $# -ne 2 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch -d "$2" -f --no-backup-if-mismatch -p$PATCHLEVEL < $0;;
    -unpatch) patch -d "$2" -f --no-backup-if-mismatch -R -p$PATCHLEVEL < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# append the patch here and adjust the -p? flag in the patch calls.

2004-04-17  Petter Reinholdtsen  <pere@hungry.com>

	* locales/et_EE: Change default charset for et_EE locale from
	ISO-8859-1 to ISO-8859-15, to reflect the content of
	<URL:http://www.eki.ee/itstandard/contents.html>.  Based on input
	from Indrek Hein and Meelis Roos.
	* SUPPORTED: Add et_EE.ISO-8859-15.

Index: localedata/SUPPORTED
===================================================================
RCS file: /cvs/glibc/libc/localedata/SUPPORTED,v
retrieving revision 1.68
diff -u -3 -p -u -r1.68 SUPPORTED
--- localedata/SUPPORTED	19 Jan 2004 23:51:37 -0000	1.68
+++ localedata/SUPPORTED	17 Apr 2004 18:10:04 -0000
@@ -161,6 +161,7 @@
 es_VE/ISO-8859-1 \
 et_EE.UTF-8/UTF-8 \
 et_EE/ISO-8859-1 \
+et_EE.ISO-8859-15/ISO-8859-15 \
 eu_ES.UTF-8/UTF-8 \
 eu_ES.UTF-8@euro/UTF-8 \
 eu_ES/ISO-8859-1 \
Index: localedata/locales/et_EE
===================================================================
RCS file: /cvs/glibc/libc/localedata/locales/et_EE,v
retrieving revision 1.11
diff -u -3 -p -u -r1.11 et_EE
--- localedata/locales/et_EE	6 Dec 2003 07:40:37 -0000	1.11
+++ localedata/locales/et_EE	17 Apr 2004 18:10:05 -0000
@@ -17,7 +17,7 @@ comment_char %
 % Application: general
 % Users: general
 % Repertoiremap: mnemonic.ds
-% Charset: ISO-8859-1
+% Charset: ISO-8859-15
 % Distribution and use is free, also
 % for commercial purposes.
 
